style.css

```diff
@@ -0,0 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+/*
+        * this style is my primary style
+            it will include
+            primary color,
+            d-noen,
+            d-flex,
+            justify-content-center,
+            justify-content-between,
+            justify-content-around,
+            weight--700,
+            weight--400,
+            weight--500,
+            align-items-center,
+            align-self-center,
+            align-content-center
+        */
```

Comment:
- [ ] KInldy consider deleting these comments lines of code for they make your codebase looks ugly and it's not a good practice for a clean code.